adoc/admin-monitoring-stack.adoc

```diff
@@ -596,6 +643,360 @@ continues with definition of dashboard JSON
 kubectl apply -f grafana-dashboards-caasp-cluster.yaml
 ----
 
+=== Installation For Subpaths
+
+[NOTE]
+====
+This installation example shows how to install and configure Prometheus and Grafana using subpaths such as example.com/prometheus, example.com/alertmanager, and example.com/grafana.
+====
+
+[IMPORTANT]
+====
+Overlapped instructions from subdomains will be omitted. Refer to the instruction from subdomains.
+====
+
+==== Prometheus
+
+. Create a configuration file `prometheus-config-values.yaml`
++
+We need to configure the storage for our deployment.
+Choose among the options and uncomment the line in the config file.
+In production environments you must configure persistent storage.
+
+** Use an existing `PersistentVolumeClaim`
+** Use a `StorageClass` (preferred)
+** Disable ingresses
+** Add the external url at which the server can be accessed
++
+----
+# Alertmanager configuration
+alertmanager:
+  enabled: true
+  ingress:
+    enabled: false
+  persistentVolume:
+    enabled: true
+    ## Use a StorageClass
+    storageClass: my-storage-class
+    ## Create a PersistentVolumeClaim of 2Gi
+    size: 2Gi
+    ## Use an existing PersistentVolumeClaim (my-pvc)
+    #existingClaim: my-pvc
+
+## Alertmanager is configured through alertmanager.yml. This file and any others
+## listed in alertmanagerFiles will be mounted into the alertmanager pod.
+## See configuration options https://prometheus.io/docs/alerting/configuration/
+#alertmanagerFiles:
+#  alertmanager.yml:
+
+# Create a specific service account
+serviceAccounts:
+  nodeExporter:
+    name: prometheus-node-exporter
+
+# Allow scheduling of node-exporter on master nodes
+nodeExporter:
+  hostNetwork: false
+  hostPID: false
+  podSecurityPolicy:
+    enabled: true
+    annotations:
+      apparmor.security.beta.kubernetes.io/allowedProfileNames: runtime/default
+      apparmor.security.beta.kubernetes.io/defaultProfileName: runtime/default
+      seccomp.security.alpha.kubernetes.io/allowedProfileNames: runtime/default
+      seccomp.security.alpha.kubernetes.io/defaultProfileName: runtime/default
+  tolerations:
+    - key: node-role.kubernetes.io/master
+      operator: Exists
+      effect: NoSchedule
```

Comment:
I'm a little confused by the comment, and maybe I'm just not familiar with the syntax.  The comment says this will allow scheduling node-exporter on master nodes, but the toleration seems to say if the key exists then NoSchedule...